src/provider_misc.jl

```diff
@@ -12,6 +12,18 @@ const serverCapabilities = ServerCapabilities(
 
 function process(r::JSONRPC.Request{Val{Symbol("initialize")},Dict{String,Any}}, server)
     server.rootPath=haskey(r.params,"rootPath") ? r.params["rootPath"] : ""
+    if server.rootPath!=""
+        for (root, dirs, files) in walkdir(server.rootPath)
+            for file in files
+                if splitext(file)[2]==".jl"
+                    filepath = joinpath(root, file)
+                    uri = string("file:///", replace(replace(filepath, '\\', '/'), ":", "%3A"))
```

Comment:
linux (and MacOS?) already use forward slashes so we need something along the lines of: 
`uri = is_windows() ? string("file:///", replace(replace(filepath, '\\', '/'), ":", "%3A")) : string("file://", filepath)
            println(uri)`